UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs

```diff
@@ -1047,18 +1044,29 @@ public void CMDTryEscapeContainer(MoveAction moveAction)
 		}
 	}
 
+	public void HandleFootstepLogic()
+	{
+		if (RNG.Random.Next(0, 1000 + 1) == 1000)
```

Comment:
why do chances like this and not using the ``DMMath.Prob`` method instead? That one takes an int so you can directly type the percentage of how likely something is to happen

Reply:
`Random.Next` is also slightly more expensive because it has to create an array in the background to generate the random number.